app/tasks/mailchimp.py

```diff
@@ -70,6 +71,13 @@ def create_campaign(client, recipients, settings=None):
         rollbar.report_exc_info()
         raise e
```

Comment:
What happens if we receive a `RequestException`?  Should we publish here as well?

Reply:
It has been added.